src/views/ModelPlan/TaskList/OpsEvalAndLearning/Evaluation/index.tsx

```diff
@@ -260,183 +259,138 @@ const Evaluation = () => {
                     scrollElement="evaluationApproaches"
                     error={!!flatErrors.evaluationApproaches}
                   >
-                    <FieldArray
-                      name="evaluationApproaches"
-                      render={arrayHelpers => (
-                        <>
-                          <Label
-                            htmlFor="ops-eval-and-learning-evaluation-approac"
-                            id="ops-eval-and-learning-evaluation-approac"
-                            className="maxw-none"
-                          >
-                            {opsEvalAndLearningT('evaluationApproaches.label')}
-                          </Label>
-
-                          {itSolutionsStarted && (
-                            <ITSolutionsWarning
-                              id="ops-eval-and-learning-evaluation-approach-warning"
-                              onClick={() =>
-                                handleFormSubmit(
-                                  `/models/${modelID}/task-list/it-solutions`
-                                )
-                              }
-                            />
-                          )}
+                    <Label
+                      htmlFor="evaluationApproaches"
```

Comment:
`htmlFor` is meant to reference the `id` of the input, not the gql/formik value.  It should be `ops-eval-and-learning-evaluation-approach`